docs/guides/get-qpu-information.ipynb

```diff
@@ -44,7 +44,11 @@
    "source": [
     "## List backends\n",
     "\n",
-    "To view the backends you have access to, you can either view a list on the [Compute resources page,](https://quantum.cloud.ibm.com/computers) or you can use the [`QiskitRuntimeService.backends()`](../api/qiskit-ibm-runtime/qiskit-runtime-service#backends) method. This method returns a list of [`IBMBackend`](../api/qiskit-ibm-runtime/ibm-backend) instances:"
+    "To view the backends you have access to, you can either view a list on the [Compute resources page,](https://quantum.cloud.ibm.com/computers) or you can use the [`QiskitRuntimeService.backends()`](../api/qiskit-ibm-runtime/qiskit-runtime-service#backends) method. This method returns a list of [`IBMBackend`](../api/qiskit-ibm-runtime/ibm-backend) instances:\n",
+    "\n",
+    "<Admonition type=\"note\">\n",
+    "If you are logged in to a specific instance, only the backends available to you on that instance are returned.\n",
```

Comment:
This applies to the how you initialize your `QiskitRuntimeService` as well. For example, if you do `QiskitRuntimeService(instance=...)`, then `QiskitRuntimeService.backends()` would return only backends that can be accessed via that instance (with or without filters).